database/schema.ts

```diff
@@ -292,6 +292,38 @@ export const purchasesTable = mysqlTable('purchases', {
     .notNull(),
 });
 
+export const bannerJobTable = mysqlTable('banner_jobs', {
+  id: bigint('id', {
+    unsigned: true,
+    mode: 'number',
+  })
+    .autoincrement()
+    .primaryKey(),
+  user_id: mediumint('user_id', {
+    unsigned: true,
+  })
+    .references(() => userTable.id, {
+      onDelete: 'cascade',
+    })
+    .notNull(),
+  username: varchar('username', {
+    length: 32,
+  }).notNull(),
```

Comment:
I was going to flag this because we have the user id, but then I realised it's legitimate they can change their username and we want to log the data "as is". so I think you're right here, nice job